### PR TITLE
chore: add maintainers as codeowners to all files

### LINKED ISSUE #166: Add Repo Maintainers to CODEOWNERS file
# Update CODEOWNERS to provide maintainers ownership of all files in their repos

**Description**:

Update this repositories `.github/CODEOWNER` file to ensure that the repository maintainer
group(s) have full control of the files in this repository.

### PR DESCRIPTION
**Pull request by PavelSBorisov**
**Description**:
Update CODEOWNERS to provide maintainers ownership of all files in their repos

**Related issue(s)**:

Fixes #166 

**Notes for reviewer**:
N/A

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
 # NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below
-*                                                 @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
+*                                               @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
 
 #############################
 ##### Auth-Layer-Proxy ######
 #############################
-/auth-layer-proxy/                                @AlfredoG87 @Nana-EC @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
-/charts/auth-layer-proxy/                         @AlfredoG87 @Nana-EC @beeradb @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/auth-layer-proxy/                              @AlfredoG87 @Nana-EC @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:10]: Alfredo is in the hedera-smart-contracts team and should be removed as an individual user.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
+/charts/auth-layer-proxy/                       @AlfredoG87 @Nana-EC @beeradb @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:11]: Alfredo is in the hedera-smart-contracts team and should be removed as an individual user.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -2,23 +2,23 @@
...
 
 ###############################
 #####       Charts       ######
 ###############################
-/charts/                                          @AlfredoG87 @Nana-EC @beeradb @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/charts/                                        @AlfredoG87 @Nana-EC @beeradb @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:16]: Alfredo is in the hedera-smart-contracts team and should be removed as an individual user.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -2,23 +2,23 @@
...
 
 ###############################
 #####      Subgraphs     ######
 ###############################
-/subgraphs/                                       @AlfredoG87 @Nana-EC @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/subgraphs/                                     @AlfredoG87 @Nana-EC @hashgraph/hedera-smart-contracts @hashgraph/release-engineering @hashgraph/release-engineering-managers
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:21]: Alfredo is in the hedera-smart-contracts team and should be removed as an individual user.

**Review by andrewb1269hg — CHANGES_REQUESTED** [review]

**Review by andrewb1269hg — APPROVED** [review]

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -30,19 +30,19 @@
 /.github/                                       @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:30]:
```suggestion
/.github/                                       @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers
/.github/workflows/                             @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts 
```

**Review by rbarkerSL — CHANGES_REQUESTED** [review]

**Review by andrewb1269hg — APPROVED** [review]

**Review by rbarkerSL — APPROVED** [review]

**Review by rbarkerSL — APPROVED** [review]

**Pull request merged**